File: semester-1/bases-de-datos/petcon/web_app/models/customer.py
class Customer:
    CUSTOMER_ID = "identifier"
    FIRST_NAME = "first_name"
    LAST_NAME = "last_name"
    EMAIL = "email"
    CREATED_AT = "created_at"

    def __init__(self, identifier, first_name, last_name, email, created_at):
        self.identifier = identifier
        self.first_name = first_name
        self.last_name = last_name
        self.email = email
        self.created_at = created_at

    @staticmethod
    def create_from_json(json):
        return Customer(json[Customer.CUSTOMER_ID],
                        json[Customer.FIRST_NAME],
                        json[Customer.LAST_NAME],
                        json[Customer.EMAIL],
                        json[Customer.CREATED_AT])

File: semester-1/bases-de-datos/petcon/desktop_app/src/PETCON.DesktopApp/Models/Customer/Customer.cs
using System;

namespace PETCON.DesktopApp.Models.Customer
{
    public class Customer
    {
        public Guid Identifier { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Email { get; set; }

        public DateTime CreatedAt { get; set; }

        public bool IsActive { get; set; }
    }
}